### PR TITLE
Feature/discount docs

### PR DESCRIPTION
**Pull request by nk-coding**
https://frontend.gropius.duckdns.org/projects/e0f96bfe-7221-4bd8-b665-84ff023f80f9/issues/4b0fa624-e70a-4486-a78b-1573852f5129

Note: does not include rendered GraphQL docs yet
Edit: also adds a few fixes to shipment docs because I am lazy

DoD:
- [x] Requirements of the issue are met
- [x] Code has been reviewed
- [x] Code is documented
- [x] GraphQL related artefacts are documented

### CODE REVIEW

File: docs/docs/dev-manuals/services/discount.mdx [hunk 1]
```diff
@@ -0,0 +1,225 @@
+import CustomImage from "@site/src/components/CustomImage";
+
+# Discount Service
+
+The discount service provides the bounded context `Discount`.
+It is responsible for managing discounts, coupons, and their usages by userso.
```
Comment (lesser1) [inline docs/docs/dev-manuals/services/discount.mdx]: Typo in "userso"?!

File: docs/docs/dev-manuals/services/shipment.mdx [hunk 2]
```diff
@@ -93,6 +141,8 @@ enum ShipmentStatus {
 }
 ```
 
+````
```
Comment (lesser1) [inline docs/docs/dev-manuals/services/shipment.mdx]: I think this shouldn't be here...
This and line 162 cause the section to be formatted as some sort of code block.

File: docs/docs/dev-manuals/services/shipment.mdx [hunk 3]
```diff
@@ -109,7 +159,7 @@ This event is published when a new shipment is created.
   shipmentMethodId: string,
   shipmentAddressId: string
 }
-```
+````
```
Comment (lesser1) [inline docs/docs/dev-manuals/services/shipment.mdx]: See previous comment.

**Review by lesser1 — COMMENTED** [review]
Overall, it looks good, just some minor things to take a second look at:  a typo and formatting.

File: docs/docs/dev-manuals/services/discount.mdx (hunk 1 above)
Comment (nk-coding) [reply docs/docs/dev-manuals/services/discount.mdx]: fixed with 5cb6c4b68ae5261f254cdc5392f3bf7c0a1560e7

File: docs/docs/dev-manuals/services/shipment.mdx (hunk 2 above)
Comment (nk-coding) [reply docs/docs/dev-manuals/services/shipment.mdx]: fixed with 5cb6c4b68ae5261f254cdc5392f3bf7c0a1560e7

File: docs/docs/dev-manuals/services/shipment.mdx (hunk 3 above)
Comment (nk-coding) [reply docs/docs/dev-manuals/services/shipment.mdx]: fixed with 5cb6c4b68ae5261f254cdc5392f3bf7c0a1560e7

**Review by lesser1 — APPROVED** [review]

**Pull request merged**